plugins/modules/aci_access_switch_policy_group.py

```diff
@@ -0,0 +1,472 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2023, Tim Cragg (@timcragg)
+# Copyright: (c) 2023, Akini Ross (@akinross)
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: aci_access_switch_policy_group
+short_description: Manage Access Switch Policy Groups (infra:AccNodePGrp).
```

Comment:
Should we also add infraSpineAccNodePGrp into the same module? If not, how do we name the two?

Reply:
Had a look and the spine one has a lot less options, so perhaps two modules would be better? I can rename to aci_access_leaf_switch_policy_group and create issue for the aci_access_spine_switch_policy_group?